packages/documentation/src/utils/version.ts

```diff
@@ -29,7 +29,7 @@ const versionFilterMap: any = {
 };
 
 export function getVersion(version: string, filter: string = '') {
-  const cleanVersion = DEPENCENCIES[version].replace(/^[^\d]+/, '');
+  const cleanVersion = version.replace(/^[^\d]+/, '');
```

Comment:
Not sure, but I think this breaks the version extraction:
<img width="1011" alt="image" src="https://github.com/swisspost/design-system/assets/1659006/2dab2d52-8c76-4c04-8288-cf0917e63b54">

Reply:
Good catch!